etl/src/main/java/org/pih/hivmigration/etl/sql/LabResultMigrator.groovy

```diff
@@ -0,0 +1,261 @@
+package org.pih.hivmigration.etl.sql
+
+class LabResultMigrator extends SqlMigrator {
+
+    @Override
+    def void migrate() {
+        executeMysql("Create staging table", '''
+            create table hivmigration_lab_results (
+              obs_id int PRIMARY KEY AUTO_INCREMENT,
+              source_patient_id int,
+              source_encounter_id int,
+              source_result_id int,
+              sample_id VARCHAR(20),
+              test_type VARCHAR(16), -- viral_load, CD4, tr, ppd, hematocrit
+              obs_datetime date,
+              value_numeric DOUBLE,
+              value_text VARCHAR(100),
+              value_boolean BOOLEAN,
+              vl_beyond_detectable_limit BOOLEAN,
+              vl_detectable_lower_limit DOUBLE,
+              uuid char(38),
+              KEY `source_encounter_id_idx` (`source_encounter_id`)
+            );
+        ''')
+
+        loadFromOracleToMySql('''
+            insert into hivmigration_lab_results
+               (source_patient_id,
+                source_encounter_id,
+                source_result_id,
+                sample_id,
+                test_type,
+                obs_datetime,
+                value_numeric,
+                value_text,
+                value_boolean,
+                vl_beyond_detectable_limit,
+                vl_detectable_lower_limit)
+            values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+        ''', '''
+            select   e.patient_id as source_patient_id,
+                     e.ENCOUNTER_ID as source_encounter_id, 
```

Comment:
Assumedly the source encounter ID is either type "lab_result" or "anlap_lab_result", which we are already migrating to the Lab Results encounter type in the encounter migrator?  We shoudl just double-check this, to make sure we don't get any obs that are floating without encounters.  (Which might be okay, but if we aren't aware of it and plan for it, they will likely escape into the ether)